aslam_cv_common/include/aslam/common/time.h

```diff
@@ -72,6 +72,19 @@ inline bool isValidTime(int64_t time) {
   return time != getInvalidTime();
 }
 
+inline std::string formatTimeNanoseconds(const int64_t timestamp_nanoseconds) {
```

Comment:
why is it called format time? isn't this just taking timeNanosecondsToString?

Reply:
with `timeNanosecondsToString()` I would expect the same output as `std::to_string()`, but this is not the case here. do you have another name in mind which you prefer?

Reply:
but I think this is what happens within C# and so one, toString produces a nice output with units or whatever is specific for a given type. In this case, that would be units. For me, ``format`` doesn't mean anything.

Reply:
I prefer ``timeNanosecondsToString``